### DISCUSSION TITLE
How to map struct pointer in struct?

### DISCUSSION THREAD

**Discussion by luxingxiao** [bytedeco/javacpp#642] [Q&A] [answered]
@saudet  Hi, do you know how to map struct pointer in struct to array?
 like this:
```
typedef struct DDS_XTypes_CompleteStructMemberSeq
{
  uint32_t _maximum;
  uint32_t _length;
  struct DDS_XTypes_CompleteStructMember *_buffer;
  bool _release;
} DDS_XTypes_CompleteStructMemberSeq;
```

In this case, _buffer shuld be map to DDS_XTypes_CompleteStructMember  array, but it is mapped to DDS_XTypes_CompleteStructMember  like this:
```
public native DDS_XTypes_CompleteStructMember _buffer(); public native DDS_XTypes_CompleteStructMemberSeq _buffer(DDS_XTypes_CompleteStructMember setter);
```

**Comment by saudet** [comment, marked as answer]
DDS_XTypes_CompleteStructMember is a Pointer, that's fine. What's the issue?

**Reply by luxingxiao** [reply]
I am sorry. I ignore this. Thank you very much 😂
